File: java_python/Calculadora.java


public class Calculadora {
    private int n1;
    private int n2;

    public Calculadora(int n1, int n2) {
        this.n1 = n1;
        this.n2 = n2;
    }

    public int soma() {
        return this.n1 + this.n2;
    }

    public int subtracao() {
        return this.n1 - this.n2;
    }
}

File: java_python/Calculadora.py
import jpype

jpype.startJVM(jpype.getDefaultJVMPath(), "-Djava.class.path=.")

Calculadora = jpype.JClass("Calculadora")
n1 = int(input("Digite o numero 1: "))
n2 = int(input("Digite o numero 2: "))

calc = Calculadora(n1, n2)
print("Soma: ", calc.soma())

jpype.shutdownJVM()